testinput_tier_1/inputs/ufs_c48/INPUT/coupler.res.correct

```diff
@@ -0,0 +1,3 @@
+     3        (Calendar: no_calendar=0, thirty_day_months=1, julian=2, gregorian=3, noleap=4)
```

Comment:
Note to self. When we fix the code to read the current time correctly from coupler.res, we can replace the current/hacked coupler.res with this (correct) one.